subprojects/cmocka.wrap

```diff
@@ -0,0 +1,13 @@
+[wrap-file]
+directory = cmocka-1.1.7
+source_url = https://cmocka.org/files/1.1/cmocka-1.1.7.tar.xz
+source_filename = cmocka-1.1.7.tar.xz
+source_hash = 810570eb0b8d64804331f82b29ff47c790ce9cd6b163e98d47a4807047ecad82
+patch_filename = cmocka_1.1.7-3_patch.zip
+patch_url = https://wrapdb.mesonbuild.com/v2/cmocka_1.1.7-3/get_patch
+patch_hash = 527c16d9819a69e69e807eadb4d101cc93b99858120da239da07208562a83d55
+source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/cmocka_1.1.7-3/cmocka-1.1.7.tar.xz
+wrapdb_version = 1.1.7-3
+
+[provide]
+cmocka = cmocka_dep
```

Comment:
missing new line

Reply:
what is fun here.... is that thils file is the perfect copy of the wrapdb file (just DL :p). Fixing that :)